useScroll/useScrollbar.js

```diff
@@ -4,11 +4,11 @@ import {constants} from '@enact/ui/useScroll';
 const {paginationPageMultiplier} = constants;
 
 const useScrollbar = (props, instances, context) => {
```

Comment:
One concern is that Sandstone doesn't have a useScrollbar.js, this feature moved into useScroll.js. I understand that Agate useScrollbar has more feature because it has a PageControll button. I hope to revisit to check we don't need to unify the structure of Sandstone and Agate.